### ISSUE TITLE
how to obtain bot token ? 

### ISSUE THREAD

**Issue by tigercoding56** [Baggette/minecraft-server-ping-example#3]
there is probably some kind of API where you register it but insufficient instructions , also i suggest to  find a solution that does not require a 3rd party API (for instance by attempting to send log in request to server using deliberately wrong info (it will not connect but if you get a response server is up  more info on minecraft protocol is here  https://wiki.vg/Protocol#Handshaking 
also please close some duplicate issues :

https://github.com/epeth0mus/Fugu16/issues/1
https://github.com/epeth0mus/Fugu16/issues/5
https://github.com/epeth0mus/Fugu16/issues/4

**Comment by Baggette** [comment]
> there is probably some kind of API where you register it but insufficient instructions , also i suggest to find a solution that does not require a 3rd party API (for instance by attempting to send log in request to server using deliberately wrong info (it will not connect but if you get a response server is up more info on minecraft protocol is here https://wiki.vg/Protocol#Handshaking also please close some duplicate issues :
> 
> [epeth0mus/Fugu16#1](https://github.com/epeth0mus/Fugu16/issues/1) [epeth0mus/Fugu16#5](https://github.com/epeth0mus/Fugu16/issues/5) [epeth0mus/Fugu16#4](https://github.com/epeth0mus/Fugu16/issues/4)

No, I will not be closing those issues, mr "shit0mus" post a bunch of fake jbs that literally do not work and are just forks of fugu 14 renamed to fugu 16 and the proof is literally in the commit history 
![image](https://github.com/user-attachments/assets/a843fd45-9478-4f58-8aab-8840093f70b1)
![image](https://github.com/user-attachments/assets/484d5bc0-4ab4-47f1-bc43-754435691721)

this so called "fugu 16" isn't the only project he has done this with.

**Comment by Baggette** [comment]
> there is probably some kind of API where you register it but insufficient instructions , also i suggest to find a solution that does not require a 3rd party API (for instance by attempting to send log in request to server using deliberately wrong info (it will not connect but if you get a response server is up more info on minecraft protocol is here https://wiki.vg/Protocol#Handshaking also please close some duplicate issues :
> 
> [epeth0mus/Fugu16#1](https://github.com/epeth0mus/Fugu16/issues/1) [epeth0mus/Fugu16#5](https://github.com/epeth0mus/Fugu16/issues/5) [epeth0mus/Fugu16#4](https://github.com/epeth0mus/Fugu16/issues/4)

to obtain a bot token you must go to the discord developer portal and create a new bot and get the token from there
